homeassistant/components/rki_covid/sensor.py

```diff
@@ -0,0 +1,212 @@
+"""RKI Covid numbers sensor."""
+
+from datetime import datetime, timedelta
+import logging
+from typing import Callable, Dict, Optional
+
+from rki_covid_parser.parser import RkiCovidParser
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.components.sensor import PLATFORM_SCHEMA, STATE_CLASS_MEASUREMENT
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers import update_coordinator
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import (
+    ConfigType,
+    DiscoveryInfoType,
+    HomeAssistantType,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import get_coordinator
+from .const import (
+    ATTR_COUNTY,
+    ATTRIBUTION,
+    CONF_BASEURL,
+    CONF_DISTRICT_NAME,
+    CONF_DISTRICTS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+# wait for x minutes after restart, before refreshing
+SCAN_INTERVAL = timedelta(minutes=10)
+
+# schema for each config entry
+DISTRICT_SCHEMA = vol.Schema({vol.Required(CONF_NAME): cv.string})
+
+# schema for each platform sensor
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Remove. Could you perhaps try to remove all unrelated / YAML setup parts? This will make reviewing a lot easier and faster.

Reply:
done